stdlib/public/core/Integers.swift.gyb

```diff
@@ -2204,6 +2206,56 @@ ${assignmentOperatorComment(x.operator, False)}
 }
 
 //===----------------------------------------------------------------------===//
+//===--- DefaultWords -----------------------------------------------------===//
+//===----------------------------------------------------------------------===//
+
+/// A collection containing the words of the binary representation
+/// of an arbitrary fixed-width integer type, in order from the least
+/// significant to most significant.
+///
+/// The words of negative values are in two's complement representation,
+/// regardless of the integer type's underlying implementation.
+public struct DefaultWords<Integer: FixedWidthInteger> : RandomAccessCollection {
```

Comment:
`IntegerWordsCollection` perhaps? `Default` really only makes sense if we have more than one, which we don't. Or simply `IntegerWords<Storage>`?